debug/rsp-server.c

```diff
@@ -2150,8 +2150,10 @@ rsp_vpkt (struct rsp_buf *buf)
     }
   else
     {
-      fprintf (stderr, "Warning: Unknown RSP 'v' packet type %s: ignored\n",
-	       buf->data);
+      /* GDB expects an empty response for unknown 'v' packets. */
+      if (0 != strcmp ("vMustReplyEmpty", buf->data))
```

Comment:
This looks good to me.
Could you put this in curly brackets?  It seems consistent with our other if statements even with one line.
```c
if (...)
  {
    fprintf ...
  }
```